testsuite/MDAnalysisTests/coordinates/test_tng.py

```diff
@@ -0,0 +1,186 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+import numpy as np
+import pytest
+
+from numpy.testing import (
+    assert_equal,
+    assert_allclose
+)
+import MDAnalysis as mda
+from MDAnalysis.lib.mdamath import triclinic_box
+from MDAnalysis.coordinates.TNG import HAS_PYTNG
+if HAS_PYTNG:
+    import pytng
+
+from MDAnalysisTests.datafiles import (TNG_traj, TNG_traj_gro)
+
+
+@pytest.mark.skipif(not HAS_PYTNG, reason="pytng not installed")
+class TestTNGTraj(object):
```

Comment:
Why are we not using it multiframereader base test class in the tests here? Given we have the issue opened to implement everything using them we should make sure we have a good reason to have new tests that don't use them (and document why)

Reply:
Fixed now.